Examples/Examples/FeatureFormExampleView.swift

```diff
@@ -153,15 +160,17 @@ class Model: ObservableObject {
     
     /// Submit the changes made to the form.
     func submitChanges() async {
-        guard let feature = featureForm?.feature,
-              let table = feature.table as? ServiceFeatureTable,
+        guard let featureForm = featureForm,
+              let table = featureForm.feature.table as? ServiceFeatureTable,
               table.isEditable,
               let database = table.serviceGeodatabase else {
             print("A precondition to submit the changes wasn't met.")
             return
         }
         
-        try? await table.update(feature)
+        guard featureForm.validationErrors.isEmpty else { return }
```

Comment:
For the future, we should consider showing an alert specifying how many validation errors there are, similar to FieldMaps:

![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/3998072/25edcc93-9941-410c-9667-81ddc38d844c)

Reply:
Created Apollo 388